fixed parameter object in OpenAPI is not required

rparree opened the pull request:
See https://swagger.io/specification/#parameter-object

adamw commented:
Can you maybe add a test which would show the desired generated yaml on an example endpoint? I think `VerifyYamlTest` would be a good place to put such a test

rparree commented:
i changed  the `genre` parameter of `/books/{genre}/{year}`  in `myBookshopYaml` and updated the test accordingly.  All the other fields kept the `required` field. Is that sufficient for the test?

adamw commented:
Sorry that's the codegen project, not the serializer 🤦 My bad :) This looks good then :)
